Command/BaseBootstrapSymlinkCommand.php

```diff
@@ -62,7 +62,11 @@ public static function checkSymlink($symlinkTarget, $symlinkName, $forceSymlink
                 if (!$forceSymlink) {
                     throw new \Exception(sprintf('Symlink "%s" points to "%s" instead of "%s"', $symlinkName, $linkTarget, $symlinkTarget));
                 }
-                unlink($symlinkName);
+                if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' && is_dir($linkTarget)) {
```

Comment:
As mentioned in #1059 Symfony uses the following condition for this check since 2.3.24: 
```php
if ('\\' === DIRECTORY_SEPARATOR && is_dir($linkTarget)) {
```

For consistency, we should do it that way, too.